core/whistle_apps-1.0.0/priv/couchdb/views/search.json

```diff
@@ -4,6 +4,12 @@
     "views": {
         "search_by_name": {
             "map": "function(doc) { if (doc.pvt_deleted) return; for (i in doc.pvt_tree) { emit([doc.pvt_tree[i], doc.pvt_type, doc.name], {'id': doc._id, 'descendants_count': doc.descendants_count || 0, 'name': doc.name, 'realm' : doc.realm}); } }"
+        },
```

Comment:
How do you compile Crossbar? The Makefile should run a JSON formatter, which puts `search_by_name` below these two new views (it sorts the keys alphabetically). When I compile this branch, I have an unstaged change in this file from the JSON formatter running. You may want to check your build process to make sure the JSON formatter is successfully running (and commit the formatted `search.json` as evidence).